src/user/commands/edit-user.command.ts

```diff
@@ -109,7 +109,7 @@ export class EditUserCommand {
    * *********************************************************************************
    */
   private verifyUserNameField(data: UserEditDTO) {
-    if (data.name) {
+    if (data.name !== undefined) {
```

Comment:
Faz assim que fica mais genérico:

```suggestion
    if (!!data.name) {
```